### ISSUE TITLE
Fatal error in class-modules-repository.php

### ISSUE THREAD

**Issue by nielsvr** [radishconcepts/woocommerce-naguro-connect#98]
It seems that something is timing out there, throwing a WP_Error into the json_decode and causing a Fatal error:

Fatal error: Cannot use object of type WP_Error as array in /wp-content/plugins/woocommerce-naguro-connect/includes/naguro/repositories/class-modules-repository.php on line 77

**Comment by coenjacobs** [comment]
Yeah, our API had a bad day. Sorry about that. These errors need to be caught though, so leaving this issue open to fix in the upcoming fix release.
